packages/components/psammead-brand/src/index.jsx

```diff
@@ -131,7 +131,7 @@ const LocalisedBrandName = ({ linkId, product, serviceLocalisedName }) =>
       <span>{serviceLocalisedName}</span>
     </VisuallyHiddenText>
   ) : (
-    <VisuallyHiddenText>{product}</VisuallyHiddenText>
+    <VisuallyHiddenText id={`BrandLink-${linkId}`}>{product}</VisuallyHiddenText>
```

Comment:
maybe this should be a variable in case we change this one but not the other id

Reply:
👍 Yes you are right